app/services/request_password_reset.rb

```diff
@@ -16,15 +17,25 @@ def perform
   private
 
   def send_reset_password_instructions
+    irs_params = {
+      email: email,
+    }
+
     if Throttle.new(user: user, throttle_type: :reset_password_email).throttled_else_increment?
       analytics.throttler_rate_limit_triggered(throttle_type: :reset_password_email)
+      irs_params[:success] = false
+      irs_params[:failure_reason] = { throttled: 'Rate limit triggered for the user.' }
```

Comment:
Also worth noting if this formatting is sufficient.

Reply:
This will most likely be split into a separate event to simplify contracts / expected payloads.

Reply:
Split the event into LG-7335